search-relevancy-test/src/search_relevancy_test/anomaly_fetcher.clj

```diff
@@ -48,6 +50,8 @@
   [concept-id fmt]
   (let [url (format "%s/concepts/%s" (source-url) concept-id)
         params {:query-params {:concept-id concept-id}}
+                ;; Uncomment when collections are only visible for a certain token
+                ; :headers {:echo-token (System/getenv "CMR_SYSTEM_TOKEN")}}
```

Comment:
Can we just always pass the token, that way you wouldn't have to remember to uncomment it out?

Reply:
If we do keep this commented out, it should be two semi-colons (per the style guide; end-of-line comments are the only ones in the Lisp world that get one semi-colon).

Reply:
Fixed to use a token if configuration is set and not pass in the header otherwise.